src/candidates/sign/router.js

```diff
@@ -25,20 +27,23 @@ export const SIGN_ROUTER = {
       path: 'sign/success-registration',
       component: SignMessage,
       beforeEnter: (to, from, next) => {
-        if (!localStorage.getItem('successRegistration')) {
```

Comment:
`grep` found no instances in the code base of the `successRegistration` item ever being set. I assumed this was not implemented. The success branch would thus always fire.